rules/linux/auditd/lnx_auditd_find_cred_in_files.yml

```diff
@@ -16,11 +16,12 @@ logsource:
 detection:
     selection:
         type: 'EXECVE'
-    keywords:
-        '|all':
-            - 'grep'
-            - 'password'
-    condition: selection and keywords
+        a0: 'grep'
+        a1: '--color=auto' # Is Alias
+        a2: '-ri'
+        a3: 'password'
+        a4: '/'
```

Comment:
This does not fix the logic but make it more strict and will not match different orders of args.

The reason you did not get a match is probably due to a backend issue when converting the rule. Because a keyword rule should match all strings present on the log.

Since both the original and change contain the required strings, this does not improve the detections but makes it compatible with auditd. But as I said you are now hardcoding arg order.

So the change does not make sense. Please explain your reasoning?